src/components/inputs/rich-text-input/rich-text-input.js

```diff
@@ -21,17 +25,36 @@ class RichTextInput extends React.PureComponent {
     this.props.onChange(event);
   };
 
+  onBlur = (event, editor, next) => {
+    next();
```

Comment:
these are the fixes for the onBlur / onFocus.

Reply:
maybe referencing the issue for why we are using `setTimeout` is going to be helpful 🤔